develop-docs/sdk/javascript/browser-tracing.mdx

```diff
@@ -0,0 +1,151 @@
+
+![Trace Lifetime](page-based-traces.png)
+
+This tracing model has a couple of consequences that are important to understand:
+
+- **Increased Context**: Let's start with the good stuff: Since the trace id remains constant for all events on a page or route, all events within one page or route belong to the same trace.
+  This makes it easier to understand what happened prior to the error or span event users are looking at.
+- **Multiple Root Spans**: A trace can have multiple root spans or events. For instance, if users enabled interaction spans or manually create spans, they will be associated with the same trace id of the prior pageload or navigation.
+- **Quota Management**: Since the trace state does not change throughout a single page or route, the sampling decision of the initial span is carried over for all subsequent events. This means that the sampling decision is only made once per page or route and is not reevaluated for each event. Furthermore, subsequent http (`fetch` or `XmlHttpRequest`) also propagate this sampling decision to potential downstream services. This can lead to a higher number of events being sent to Sentry than before.
+- **Long-lived Traces**: Not every web application features navigations (hard or soft). For pages where users simply remain on one page without making navigations (e.g. a chat application), the trace id will remain constant for the entire duration of the user journey. This can lead to very long-lived traces that can potentially grow very large.
+
+<Note>
+
+As outlined above, some of the consequences have negative product or UX implications.
+This is a known issue and we're working on an update to the trace lifetime model to address these issues.
+However, this is a complex topic and we're still in the process of evaluating the best solution.
+
+</Note>
+
+## Trace Propagation
+
+The browser JS SDKs propagate traces like other SDKs by attaching headers to outgoing requests made with the `fetch` or `XmlHttpRequest` APIs.
+However, we can't by default add the headers to all outgoing requests as downstream services might send error responses if their CORS policies don't allow arbitrary headers.
+Therefore, the browser SDKs only attach headers to same-origin requests by default. Users have to opt into cross-origin tracing by setting the `tracePropagationTargets` option in the SDK configuration.
+
+## Trace Continuation
+
+The browser SDKs are able to continue a trace that was started on the server when the initial HTML page response is served.
+For this to work, the server-side SDK has to render `<meta>` tags containing the `sentry-trace` and `baggage` tracing data into the HTML response.
+These meta tags are picked up by the browser SDKs when it initializes (`Sentry.init`).
+
+## Tracing Without performance
+
+The browser SDKs support tracing without performance (TwP). This means that the SDK attaches tracing headers to outgoing requests but does not collect any spans.
```

Comment:
Maybe define shortly why one would do that - how does this information look like in the UI, when there are no spans but you still have tracing?